### PR TITLE
Company ID tool

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Documentation for the company ID tool

## Changes

Change the company ID topic
Downloads

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

**Review by mh-concordium — CHANGES_REQUESTED** [review]

File: source/mainnet/net/guides/company-identities.rst [hunk 1]
```diff
@@ -9,202 +9,99 @@ Company identity creation
...
+                  "0": {
+                     "signKey": "1e16c2e2302023fc5235c60734981a2427004f95b6ace50a1d8a205ee9e5f9e7",
+                     "verifyKey": "7e9983b292cf5e5822b48dbed1c2d498aca97c097f7116511f7dcf6187d218c4"
+                  }
+               },
+               "threshold": 1
+               }
+            },
+            "threshold": 1
+         }
```
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/guides/company-identities.rst:80]: This is an old code example. We should probably have a new one. I will replace it with mine.

[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/guides/company-identities.rst:80]: I have changed it. Please review. I also removed the part about encrypted transfers.

File: source/mainnet/net/guides/company-identities.rst [hunk 2]
```diff
@@ -9,202 +9,110 @@ Company identity creation
...
 Import created accounts into ``concordium-client``
-==================================================
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To use the accounts created in the Concordium Company ID tool, you must use Concordium Client. You can download it :ref:`here<concordium-node-and-client-download>`. For information about how to use it, see :ref:`concordium-client`.
 
 The account keys are primarily meant for clients to integrate into their key management solution and their software, e.g., an exchange integrating their trading platform with the Concordium chain.
 
-However if the ``account-keys.json`` file is not encrypted it can be imported into ``concordium-client`` with the command:
+However, if the ``account-keys.json`` file is not encrypted it can be imported into ``concordium-client`` with the command:
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:115]: In the command-line below, you should remove the `--format=genesis`. It is not longer needed (and also cannot be there).

Comment (abizjak) [reply source/mainnet/net/guides/company-identities.rst:115]: And in the line below that one remove the fact that `--name` is optional. It is mandatory now.

File: source/mainnet/net/guides/company-identities.rst [hunk 3]
```diff
@@ -216,41 +124,28 @@ If the account-keys.json file is encrypted then it must first be decrypted. This
```
Comment (abizjak) [inline source/mainnet/net/guides/company-identities.rst:124]: The account-keys file can no longer be encrypted either.

**Review by abizjak — APPROVED** [review]
Looks good. Just a few leftovers from the old flow that need to be cleaned up.

**Review by mh-concordium — APPROVED** [review]
Looks good.

**Pull request merged**